ext/php8/request_hooks.c

```diff
@@ -132,7 +132,16 @@ void dd_request_init_hook_rinit(TSRMLS_D) {
     }
 
     zval exists_flag;
-    php_stat(DDTRACE_G(request_init_hook), strlen(DDTRACE_G(request_init_hook)), FS_EXISTS, &exists_flag TSRMLS_CC);
+#if PHP_VERSION_ID >= 80100
```

Comment:
I think this is supposed to be:

```suggestion
#if PHP_VERSION_ID < 80100
```